README.md

```diff
@@ -97,12 +97,22 @@ var sleepMs = configWrapper.Get<int>("sleep-time-in-ms", 1000);
 Loads from the Windows registry, defining a root key for my application.
 
 ``` 
-IWritableConfigWrapper = new WindowsRegistryConfigWrapper("HKLM/MyApplication/");
+IWritableConfigWrapper = new WindowsRegistryConfigWrapper("HKLM/Software/MyApplication/");
 configWrapper.Set<int>("MyKey", 5000); 
-var sleepMs = configWrapper.Get<int>("sleep-time-in-ms", 5000);
+var sleepMs = configWrapper.Get<int>("MyKey", 5000);
```

Comment:
This change bring my attention. I'm not quite sure here if the intention is to have a different delay by key. If so maybe add "MyKey" + "_Sleep". Not fully sure, just thinking .

Reply:
These are just examples for the read me.  The old example set MyKey and read sleep-time-in-ms which is confusing.